code/app/src/main/java/com/example/snackoverflow/add_ingredient_fragment.java

```diff
@@ -0,0 +1,125 @@
+package com.example.snackoverflow;
```

Comment:
This is a duplicate of the fragment @raquibk created. I suggest seeing if we can reuse the fragment which is already created (it makes the code more organized and less complicated). If this absolutely can't be reused, please change the name of the file.

**_Follow naming Convention as in the other files._**